recipes/hisat-3n/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "0.0.3" %}
+{% set name = "hisat-3n" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fulcrumgenomics/hisat-3n/archive/{{ version }}.tar.gz
+  sha256: 6a6c6ea7e46b8b75136fe93ea067bd77ef60baff9e7fc6dae9bbc60aec256acc
+  patches:
+    - simde-mr-c.patch
+
+build:
+  number: 0
+  run_exports: '{{ pin_subpackage(name,  max_pin="x") }}'
+  
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+  run:
+    - python >3.5
+    - perl
+
+test:
+  commands:
+    - hisat-3n --help | grep -F {{ version }}
```

todo: add all hisat-3n binaries to the list to test, since that would catch `hisat-3n-build` not working out of the box
```suggestion
    - hisat-3n --help | grep -F {{ version }}
    - hisat-3n-build --help | grep -F {{version }}
    - hisat-3n-table --help | grep -F {{version}}
```

---

`hisat-3n-table` does not have version info in its usage information so I did not include this test:
```
$ hisat-3n-table -h
hisat-3n-table developed by Yun (Leo) Zhang
Usage:
hisat-3n-table [options]* --alignments <alignmentFile> --ref <refFile> --output-name <outputFile> --base-change <char1,char2>
  <alignmentFile>           SORTED SAM filename. Please enter '-' for standard input.
  <refFile>                 reference file (should be FASTA format).
  <outputFile>              file name to save the 3n table (tsv format). By default, alignments are written to the “standard out” or “stdout” filehandle (i.e. the console).
  <chr1,chr2>               the char1 is the nucleotide converted from, the char2 is the nucleotide converted to.
Options (defaults in parentheses):
 Input:
  -u/--unique-only          only count the base which is in unique mapped reads.
  -m/--multiple-only        only count the base which is in multiple mapped reads.
  -c/--CG-only              only count CG and ignore CH in reference.
  --added-chrname           please add this option if you use --add-chrname during HISAT-3N alignment.
  --removed-chrname         please add this option if you use --remove-chrname during HISAT-3N alignment.
  -p/--threads <int>        number of threads to launch (1).
  -h/--help                 print this usage message.
```